### ISSUE TITLE
Add datetime to recordings list

### ISSUE THREAD

**Issue by maxwofford** [hackclub/slash-z#53]
<img width="1090" alt="Screen Shot 2022-04-26 at 08 44 11" src="https://user-images.githubusercontent.com/5891442/165302577-f7ebc213-1378-4fce-8bae-d68343d910e9.png">

**Comment by YodaLightsabr** [comment]
Are the times of the meetings already in the database? I can work on this if you'd like

**Comment by maxwofford** [comment]
Yep! right now they're being used to show a relative time (ex. "5 minutes ago"), but this would be to add an absolute time

**Comment by YodaLightsabr** [comment]
I think this can now be closed
